bin/run

```diff
@@ -153,7 +153,7 @@ case $1 in
     airflow upgradedb
 
     # Only init connections in dev
-    [ ! -z ${DEVELOPMENT+check} ] && init_connections && init_variables
+    #[ ! -z ${DEVELOPMENT+check} ] && init_connections && init_variables
```

Comment:
I think we should make this configurable via an environment variable in the docker-compose file, so we can still get the old behavior by setting a flag.

Reply:
Agreed